Add Brigmed

Martynas6ha4 opened the pull request:
## About the PR

Added brigmed to security, changed medkits to fit in outer suit slots, changed & added brigmed-related items
this includes:

- +loadout
- +brigmed locker teleporter (given lack of mapping)
- +new brigmed SOP 
- +medkits to outer suit slots
- +brigmed armor coat has outer suit slot
- +brigmed requires secoff hours and CMO hours (see below in balance)
- +brigmed vest (corpsman vest, recolored, paramedic contents)
- +brigmed slot to maps
- +brigmed glasses (look i  made these before they made medsecs craftable, maybe remove?)
- +brigmed backpacks have an attachable slot for a defibrillator (eject -> defibrillator)


## Why / Balance
brigmed, bridgemed, brigmedic, all synonyms for a role we yearn in our hearts.

In general i think it's a fun role when people aren't powergaming as a sec off with a roundstart hardsuit and I've tried to "balance" that as much as i could by giving brigmed a bit less medical items roundstart, giving it CMO hours as requirement to set a 'higher' class standard.

thus new SOP is there as well, with hopes that folk will actually care about it but it really doesn't say much

## Technical details
wow 2nd pull request cause i can't read

first time contributor to anything you should fucking smite me for the mess i might've made

notes;


todo:

-fix and balance

## Media
![brigmed1](https://github.com/user-attachments/assets/0d4b864d-2895-4002-8487-6a2ef9287245)
![brigmed2](https://github.com/user-attachments/assets/938eeab5-3fd6-4ffe-b4e3-e2a18ec9bf49)
![brigmed3](https://github.com/user-attachments/assets/1a3d98ae-192b-4b72-9e86-cc3ae7d1d676)


## Requirements
- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

**Changelog**
:cl:
- add: Added Brigmedic
- add: Added Brigmedic area to current maps in pool
- tweak: Medkits may now be equipped in suit slots

Gallagin commented:
you added brigmedic slots only to the wizden maps
the goobstation ones can be found at Resources\Prototypes\_Goobstation\Maps
kettle, barratry, leonid, submarine etc are contained in that folder

Martynas6ha4 commented:
> you added brigmedic slots only to the wizden maps the goobstation ones can be found at Resources Prototypes _Goobstation Maps kettle, barratry, leonid, submarine etc are contained in that folder

cheers fixed

PuroSlavKing commented:
sigma

PuroSlavKing commented on Resources/Prototypes/Catalog/Fills/Lockers/security.yml:
```diff
@@ -112,19 +113,8 @@
-      - id: MedkitToxinFilled
-        prob: 0.3
-      - id: MedkitBurnFilled
-        prob: 0.7
```
You literally took away the variety, the point was that every round, the brigmedic always has a different set of first-aid kits, and not the same one, so that life doesn't seem like honey 😞

Martynas6ha4 replied:
I think the issue is that you get too few or too many medkits. I mean if you could have 2 medkits guaranteed but they're random i'd go for that. Maybe I'll re-implement you're selling me on it.

Martynas6ha4 replied:
I've readded your variety in the form of (1) random medkit but i might adjust for it to be 1-2 random medkits

HenhenIII commented:
> Please god no. These were a catagorical failure on wizden for multiple reasons; Split Identity - Medical Doctor with sec access or Security officer with Medical access. Job Abandonment based on split identity - "Brig"medics consistently disregardeded their role and instead became a NT version of the Nukie Agent. Awful resepction in general - Became a defacto "captain is injured don't go to med go to brigmed". Denying alot of possibilities for antag actions.
> 
> I would strongly recommoned hard pushing it in the "medical doctor with sec access" route.
> 
> 1. Remove the armor or lock it away in the armory for code red etc.
> 2. Armor should be catagorically dogshit or just equal to a normal sec vest, they shouldn't be insentivised to be a combat role.
> 3. Remove the disabler from heeir locker - see 2 reasoning. (granted this is near immediately bypassed by mapped disabler safes but still)

It's impossible to avoid it. Most of the time brigmedics have nothing to do, knowing the current playerbase they don't enjoy doing nothing for an hour. The prisoners will not be beaten up all the time. Leaving most of brigmeds to probably go the "sec with med acces". This also removes half of medical gameplay, like you said
>  "captain is injured don't go to med go to brigmed"

I would like to add it makes sec go to the brigmed leaving behind normal medical, who's patients are a lot of time sec.

Since brigmedics would have no absolute gameplay without becoming a nuclear NT agent for only combat, they would mainly be looking at brig and Perma. Escaping Perma is already hard enough, not to even think about when the brigmedic walks in every 2 mins. (I don't want them to be a nuclear NT agent for only combat) 

With these major draw backs it's just easier to call in a normal medic when like only 3 prisoners are actually majorly hurt (not able to live through a sentence, near crit, ect) during a single shift.

misandrie commented:
> The prisoners will not be beaten up all the time.

failrp

Martynas6ha4 commented:
> EDIT: there was a large thread on goob why the brigmedic would be bad and a lot of disliked the idea also, can't find it tho


For some reason though i keep joining games and seeing brigmed in the crew manifest, almost like it's a fun role to play.

But i agree there will be adjustments to be made and i'll see what i can do. If people do despise it just remove it afterwards i'll do the legwork myself.

I've moved the hardsuit to the armory in all maps in pool, removed disabler from locker, among other things

> Armor should be catagorically dogshit or just equal to a normal sec vest,

It already is unless i've missed something? I think the vest has extra CAUSTIC damage which is a rare damage type anyway imo

Martynas6ha4 commented:
Map changes but i probably missed one or two, i changed all maps in current pool (+ lambda):
Fland:


![flandmap1](https://github.com/user-attachments/assets/f9486aab-8a3f-497f-bf79-1a16e094b76d)
![flandmap2](https://github.com/user-attachments/assets/8ca9e0c5-940e-4167-97e8-cca5d9780be5)

Kettle:
![kettlemap1](https://github.com/user-attachments/assets/ca86186a-b026-4ebe-b3a7-dd72576ed1d3)

Packed:
![packedmap1](https://github.com/user-attachments/assets/7004717b-d89d-46a8-8f07-234b03cbded2)

Saltern:
![salternmap1](https://github.com/user-attachments/assets/94db8da2-3e1e-481f-812b-52d5599e480d)

Origin:
![originmap1](https://github.com/user-attachments/assets/f680da33-2e40-4add-8f40-9945754dfa8e)

Omega:
![omegamap1](https://github.com/user-attachments/assets/4207e6f8-a95d-4803-a2e0-9fd71c90b0f1)

Oasis:
![oasismap1](https://github.com/user-attachments/assets/31ea40e0-79dc-437d-a812-22e0308077a7)

Meta:
![metamap1](https://github.com/user-attachments/assets/7d1fa2a2-bf85-4af5-8451-7443fdb4ff64)

Marathon:
![marathonmap1](https://github.com/user-attachments/assets/4ea69508-1a81-46ee-b3f6-8d0ec4fc4964)

Loop:
![loopmap1](https://github.com/user-attachments/assets/e4f4e862-2db5-4b4d-9fd0-7ed0b0b7ec72)

box:
![boxmap1](https://github.com/user-attachments/assets/02a57ea5-a6dd-481f-bf2d-22ee2195436f)

cluster:
![clustermap1](https://github.com/user-attachments/assets/b627344f-6efb-4f29-bcf7-2bf20fc5dc08)

bagel:
![bagelmap1](https://github.com/user-attachments/assets/7f299a74-42b4-4eaf-a8bf-4dbdb732eaae)

misandrie commented:
HE'S DOING TOO MUCH IN ONE PR SHUT IT DOWN

Martynas6ha4 commented:
> > Map changes but i probably missed one or two, i changed all maps in current pool (+ lambda)
> 
> Can I see what you did to my Leonid?

![image](https://github.com/user-attachments/assets/6397bb3a-1d84-44ac-9677-0e9d55aa0861)


forgot about pic but i changed pretty much nothing, added the brigmed spawn, moved the surgical crate to make space for brigmed locker, and added a brigmed suit storage to armory

Martynas6ha4 commented on Resources/Maps/_Goobstation/core.yml:
```diff
@@ -121232,6 +121316,13 @@
     - type: Transform
       pos: -32.5,-30.5
       parent: 2
+- proto: SpawnPointBrigmedic
+  entities:
+  - uid: 4821
+    components:
+    - type: Transform
+      pos: -18.5,24.5
+      parent: 2
 - proto: SpawnPointCaptain
   entities:
   - uid: 4578
```
I don't know what the AI is smoking i've ADDED brigmed and not touched captain nice one

Martynas6ha4 commented on Resources/Maps/_Goobstation/kettle.yml:
```diff
@@ -6667,6 +6674,11 @@ entities:
             1569: 66,-21
             1570: 67,-21
             1571: 67,-21
+        - node:
+            color: '#52B4E9FF'
+            id: WarnLineGreyscaleS
+          decals:
+            4982: 48,-16
         - node:
             color: '#D381C996'
             id: WarnLineGreyscaleS
```
shut up i deleted a fucking decal by accident and couldn't get the damn palette correctly if you squint at 5x zoom you'll notice it goddamn ai can't have shit in nanotrasen

HenhenIII commented:
> For some reason though i keep joining games and seeing brigmed in the crew manifest, almost like it's a fun role to play.

The same 4 players always becoming brigmed doesn't make it special.

I don't like it but since this is likely getting merged.
I haven't actually looked at the commits, can you atleast add a rule that doesn't allow the brigmed to just become a secoff with med acces?

Martynas6ha4 commented:
> > For some reason though i keep joining games and seeing brigmed in the crew manifest, almost like it's a fun role to play.
> 
> The same 4 players always becoming brigmed doesn't make it special.
> 
> I don't like it but since this is likely getting merged. I haven't actually looked at the commits, can you atleast add a rule that doesn't allow the brigmed to just become a secoff with med acces

There's SOP written on it but honestly i'll rewrite it more STRONGLY given a lot of folk seem to have this take.

Though personally i'd still like brigmed to BE there if it's full-blown code red or warops type of stuff but i suppose anyone can take a gun at that stage.

TurboTrackerss14 commented:
> > > For some reason though i keep joining games and seeing brigmed in the crew manifest, almost like it's a fun role to play.
> > 
> > 
> > The same 4 players always becoming brigmed doesn't make it special.
> > I don't like it but since this is likely getting merged. I haven't actually looked at the commits, can you atleast add a rule that doesn't allow the brigmed to just become a secoff with med acces
> 
> There's SOP written on it but honestly i'll rewrite it more STRONGLY given a lot of folk seem to have this take.
> 
> Though personally i'd still like brigmed to BE there if it's full-blown code red or warops type of stuff but i suppose anyone can take a gun at that stage.

Reminder that SoP is only upheld IC unless its something major. 
Unfortunately, role abandonment rule enforcement is lax as all hell, so its only a matter of time before "there is noone in brig to med, so I am sec now". 
(this exact same issue happened on wizden when it was there also) (the role just doesn't work unless you enforce it with solid rules and an active admin team to roleban people who can't not be shit)

Martynas6ha4 commented:
> Reminder that SoP is only upheld IC unless its something major. Unfortunately, role abandonment rule enforcement is lax as all hell, so its only a matter of time before "there is noone in brig to med, so I am sec now". (this exact same issue happened on wizden when it was there also) (the role just doesn't work unless you enforce it with solid rules and an active admin team to roleban people who can't not be shit)

I mean i'll take any suggestions. I'm of the opinion that it DOES work given how many people keep going to hop for it, the bounties that get posted, among other things, but i can see the issues it's just something that needs tweaking.

if it comes to it i'll take the nuclear option and make them pacifist if people are malding over secoffing but i feel that also takes away a lot form the role.

again, suggestions welcome.

Martynas6ha4 commented:
> > > Reminder that SoP is only upheld IC unless its something major. Unfortunately, role abandonment rule enforcement is lax as all hell, so its only a matter of time before "there is noone in brig to med, so I am sec now". (this exact same issue happened on wizden when it was there also) (the role just doesn't work unless you enforce it with solid rules and an active admin team to roleban people who can't not be shit)
> > 
> > 
> > I mean i'll take any suggestions. I'm of the opinion that it DOES work given how many people keep going to hop for it, the bounties that get posted, among other things, but i can see the issues it's just something that needs tweaking.
> > if it comes to it i'll take the nuclear option and make them pacifist if people are malding over secoffing but i feel that also takes away a lot form the role.
> > again, suggestions welcome.
> 
> The pacifism is a really bad idea. Sec can use batons and shoot disablers when pacified. It resolves nothing and makes the job even more ass.
> 
> My idea is to add the following this to the rules:
> 
> **"The brigmed is to heal prisoners of security with the given round start medical equipment. Brigmedic is not a security officer, meaning they are there to heal prisoners not to arrest criminals. Healing security and command isn't the brigmed's concern and should be left to the medical department or the blue shield officer (BSO). Only In a case of emergency, healing other crew members is allowed".**
> 
> If this was added word for word I would be completely fine with brigmed (so brigmed doesn't stack up on meds from medical to heal command and security, their job is to heal prisoners. Not be a nuclear agent for NT or a security officer.)

I feel like wording it like this is a step in the right direction but definitely not taken word for word.

I don't think it should completely mean taking away medical's job to heal a secoff if they're injured but to not ALLOW the brigmedic ON SITE to heal an officer is dumb.

> Only In a case of emergency, healing other crew members is allowed".

Besides, what constitutes an emergency? If there's revolutionaries, I'd argue that to be enough of an 'emergency' to let sec heal at brig, as they're almost surely about to be stabbed in the back if medical is compromised.

> , their job is to heal prisoners. Not be a nuclear agent for NT or a security officer.

On red alert and emergencies it's going to come down to this either way. If there's a heated gunfight happening I don't think the brigmed should sit in brig and do nothing, and works better as the 'backline' keeping the front up.

I feel like this makes for more interesting encounters than sec getting wiped in the first 5 seconds. Depends on the situation, but it seems like there's a better solution than explicitly excluding the brigmed to heal secoffs and command if you really believe that having a "trusted medic" is that detrimental.

HenhenIII commented:
>Besides, what constitutes an emergency? If there's revolutionaries, I'd argue that to be enough of an 'emergency' to let sec heal at brig, as they're almost surely about to be stabbed in the back if medical is compromised.

>>, their job is to heal prisoners. Not be a nuclear agent for NT or a security officer.

>On red alert and emergencies it's going to come down to this either way. If there's a heated gunfight happening I don't think the brigmed should sit in brig and do nothing, and works better as the 'backline' keeping the front up.

>I feel like this makes for more interesting encounters than sec getting wiped in the first 5 seconds. Depends on the situation, but it seems like there's a better solution than explicitly excluding the brigmed to heal secoffs and command if you really believe that having a "trusted medic" is that detrimental.

It's self explanatory, emergency means: "if they are unable to get to med" during revs yeah, when like sec is near critical that means emergency, they can't go to med, you can heal them. During nukies is an emergency. Also I really do believe the "trusted medica" mentality. Same thing happened with brig med on wizden, same thing happens with CMO right now.

HenhenIII commented:
i did the last one in a hush. i have wrote a completely clear message to add now, what should account from all your "questions":

**"The brigmedic is a medical officer meant to heal the prisoners of security (brig and perma) with the round start medical equipment. The brigmedic is not a paramedic and should mostly stay in brig, like warden. If security happens to run out of medical equipment then the brigmedic can collect some more from the medical department, otherwise over stacking on medicine (formally known as powergaming) isn't allowed. As a brigmedic you shouldn't be healing security/command, that job is left for the medical department or blue shield officer (BSO). This can be ignored in the case of an emergency.

Emergency for a brigmedic is defined as follows:
Unable to get to med - Brigmedic is allowed to heal people who are unable to go to med because they are crit/near crit. this includes command, security and normal crew.
Revolutionaries - If the medical department is confirmed to be filled with revolutionaries the brigmedic can heal non-prisoners. 
Nuclear operatives - This is self explanatory. During a nuclear threat the brigmedic is allowed to assist and heal anyone in need.
Syndes/Heretics/Lings/any other antagonist not mentioned - As above stated "Unable to get to med" is valid, otherwise the threat itself is not big enough for the brigmedic to abandon their job. There is one exception to this, if any antags get out of control then you can assist security and anyone in need. (out of control meaning that security is unable to handle the threat, this doesn't mean that captain died and you need to go looking for it and assist security, they can handle it on their own.) 
Wizard - If the wizard is a deadly threat to the station and has displayed violent actions then the brigmedic is allowed to assist crew. If the wizard hasn't portrayed any violent behaviour then the brigmedic should remain doing his normal job."**

Martynas6ha4 commented:
> As a brigmedic you shouldn't be healing security/command, that job is left for the medical department or blue shield officer (BSO). 

I'll be honest I don't think it fundamentally works. It's simply sounds dumb for an injured secoff to be there and the medic being like "sorry I cannot bandage you that's against SOP go to medical." Something else has got to cook, but I'm personally not against having a trusted medic, but maybe that's because sec keeps getting understaffed and beaten in my recent shifts. I'll let this question simmer as i fix the mapping.

Martynas6ha4 commented:
Anyone else reading this I also encourage you to leave ideas/feedback on the goob [discord](https://discord.com/channels/1202734573247795300/1346210448684617860), someone opened up a bounty and it's become a discussion for brigmed at this stage. Given how contentious it is i'm willing to listen to both sides within reason

HenhenIII commented:
> I'll be honest I don't think it fundamentally works. It's simply sounds dumb for an injured secoff to be there and the medic being like "sorry I cannot bandage you that's against SOP go to medical." Something else has got to cook, but I'm personally not against having a trusted medic, but maybe that's because sec keeps getting understaffed and beaten in my recent shifts. I'll let this question simmer as i fix the mapping.

wdym it sounds dumb? its exactly like BSO "sorry i cannot deal with the criminal, thats against SOP go call sec". theres no diffrence . also that line NEEDS to be there otherwise the hole "command/sec come to brigmed to heal". i don't get how you are not getting it? it happened on wizden, it happens right now with CMO. i don't get what you mean

Martynas6ha4 commented:
> command/sec come to brigmed to heal

I don't understand how this is such  a big issue is what i'm trying to say. Medical hates dealing with secoffs and secoffs hate waiting in medical. Command is already getting treated by CMO anyways.

Can I ask why this is so contentious? I just need a different viewpoint.

TurboTrackerss14 commented:
> > command/sec come to brigmed to heal
> 
> I don't understand how this is such a big issue is what i'm trying to say. Medical hates dealing with secoffs and secoffs hate waiting in medical. Command is already getting treated by CMO anyways.
> 
> Can I ask why this is so contentious? I just need a different viewpoint.

Because it leans into metagamey "This person physically cannot be antag" same with command ONLY ever getting healed by CMO. 
Denies any antag medical a chance to poison/roundremove/do antagthings to you.

Martynas6ha4 commented:
> > > command/sec come to brigmed to heal
> > 
> > 
> > I don't understand how this is such a big issue is what i'm trying to say. Medical hates dealing with secoffs and secoffs hate waiting in medical. Command is already getting treated by CMO anyways.
> > Can I ask why this is so contentious? I just need a different viewpoint.
> 
> Because it leans into metagamey "This person physically cannot be antag" same with command ONLY ever getting healed by CMO. Denies any antag medical a chance to poison/roundremove/do antagthings to you. Someones one chance to nab you with oppertune timing when you are low or vulnerable denied because "I go to brigmedic because they can't roll antag"

It's not like i disagree with the sentiment but sec players are low, and command gets so injured they need a body cloned or chems/cryo for limbs.


Antagonists getting one chance is unrealistic. It's goob, it's chaos, I'm personally not going around stalking command and waiting for them to show up in medical.  Yeah it's an opportunity taken away but you aren't roundremoving in medical unless you can chomp a brain fast enough after a minibomb.


Going to a sec medic if your life is in danger isn't meta. But going there every time after  a papercut definitely could be and it's a valid issue. I just don't think the way to solve it is with strict rules and SOP, nor do I think antags getting a really specific opportunity taken away holds enough ground. Especially since it seems the power balance favours them more currently.


I'm not adding this role with that mindset though, I just think It's neat to play. It really doesn't feel like it hinders others to that degree in actual play.

HenhenIII commented:
Tf are you tweaking about?
"Medical hates dealing with secoffs" Like UHH??. that's they're entire fucking job. Medica don't hate that, they want to heal.
"Sec hates waiting in med" are you so Inpatient that you can't wait for medical to overdose you with bic. For those NRP sec fucks, who cares, if they got such a big ego that they can't stand 30 seconds in med then they don't deserve to be healed. 
"Goob is chaotic" exactly, goobers will take any chance to abuse or powergame.

Stop thinking only about yourself and start looking it from other people's perspective. If there's a way, then there's people to abuse it

misandrie commented:
> Medical hates dealing with secoffs and secoffs hate waiting in medical. Command is already getting treated by CMO anyways.

pardon, state your credentials as to making this statement?

Martynas6ha4 commented:
> Stop thinking only about yourself and start looking it from other people's perspective. If there's a way, then there's people to abuse it

The reason I'm still replying to comments is because i want to understand both sides.

Re-reading the thread has made me realize that what I actually take issue with is the hard line at not treating command unless it's an _absolute_ necessity. Looking back I've changed my mind on it, but it should still be phrased to discourage healing command without outright banning it, outside of emergencies that is.

For example:
"Brigmedic's primary responsibility is the care of Prisoners and Security personnel. While they may provide immediate aid to Command in urgent situations, non-critical treatment should be handled by Medbay to ensure full medical resources and specialized care are available. Brigmedics are equipped for stabilization rather than comprehensive medical treatment."

It's minor and very similar to what was said originally, though this works better since it allows treating command, while still setting the precedent of it not being the norm. Thus allowing a brigmedic to direct command to medbay, or choose to treat them _according to the situation_ without worrying about edge cases. Command treatment should be handled by medbay, but it doesn't state that it must be.

However restricting brigmed from treating security doesn’t make sense for a medical role within security. Even if restricted, during red alert it effectively functions as a backline medic anyway - regardless of any SOP. There’s only one brigmedic per shift for the entirety of sec, It's reasonable to let brigmed stabilize a secoff and let medical deal with those that are too damaged to heal on the spot, as it's reasonable to let the brigmedic bandage an officer after a heated arrest.

> goobers will take any chance to abuse or powergame.

I won't disagree, but my opinion is that just because a role has potential to be abused isn't reason enough to not implement it. Other roles like Chem can cook bombs, Cargo could order guns, and Security occasionally breaks escalation. All can have varying degress of powergaming and abuse. I don't think brigmed fundamentally brings any more potential abuse than other roles. At a point abuse becomes a player issue, not a role issue.

> "Medical hates dealing with secoffs" Like UHH??. that's they're entire fucking job. Medica don't hate that, they want to heal.
"Sec hates waiting in med" 

> pardon, state your credentials as to making this statement?

If i remember i took this thought from the initial brigmed thread in which the role was removed.

I meant to reference the interactions that sometimes happen between the departments such as sec not letting medical heal someone propperly, med being packed, or secoffs always getting robbed when brought to.

Honestly, I was too tired at the time and yeah, looking back I could've phrased it better or not at all. I had the thought at the time that brigmed would make for more meaningful interaction between the departments.

Martynas6ha4 commented:
It's been a minute since I touched the PR I'll get to updating it tomorrow or the day past. Been busy with uni.

HenhenIII commented:
> Command treatment should be handled by medbay, but it doesn't state that it must be.

"command should be handled by medbay but it doesn't say it has to be, meaning i can still do it" wow, all of command is suddenly in brig. strict guidelines are needed for these kinds of things. im still gonna keep saying it, you not doing it doesn't mean others will not do it. 

> However restricting brigmed from treating security doesn’t make sense for a medical role within security. Even if restricted, during red alert it effectively functions as a backline medic anyway - regardless of any SOP. There’s only one brigmedic per shift for the entirety of sec, It's reasonable to let brigmed stabilize a secoff and let medical deal with those that are too damaged to heal on the spot, as it's reasonable to let the brigmedic bandage an officer after a heated arrest.

didn't i already write like 30 sentences why giving sec their personal healer is bad. from antag chances to metagaming.

> but my opinion is that just because a role has potential to be abused isn't reason enough to not implement it. Other roles like Chem can cook bombs, Cargo could order guns, and Security occasionally breaks escalation. 

what meth are you inhaling now. chem cooking bombs in a non emergency situation is bannable, cargo ordering guns for not a good reason is bannable, security breaking escalation is bannable, why should the brigmedic have a golden ticket to powergame and metagame. It makes NO GODDAMN sense, all of the things you gave as "examples" to prove your point are in fact bannable, like cargo can order guns in a case of emergency should the brigmedic able to heal non prisoners in the case of an emergency. 



_**please put some more thought and effort into your arguments. you've stated the same thing over and over with no actual "why its better this way rather than the other way" rather than "my opinion" "makes no sense".**_

Martynas6ha4 commented:
> please put some more thought and effort into your arguments.

im extraordinarily shit at words apologies

> what meth are you inhaling now.

sleep deprivation. i dont remember writing half the shit in there. 

my point is that an unrestricted brigmedic would not make as big of an impact as it's made out to be. not saying it should be unrestricted.

> strict guidelines are needed for these kinds of things. im still gonna keep saying it, you not doing it doesn't mean others will not do it.

the strict guidelines are already written in the rules. no powergaming is a rule. if somebody isn't respecting that adding sop isn't gonna make a difference.

why the fuck is command going to brigmed anyway they have BSO who's whole role is to ensure their survival. BSO gets meds and a surgery bag. Maybe i'm out of the loop but i don't see many command flocking to BSO to avoid stepping a foot in medical.

even cmo's sop encourages cmo being the one to treat command:

_"The Chief Medical Officer is expected to work on Command Personnel over regular Medical Personnel when possible."_

adding arbitrary hard restrictions preventing someone from treating does more harm than good. A hurt command member steps into sec where the medical role can't bandage them but the secoff can. preventing treatment solves nothing, and makes for an awkward in-universe exchange where the doctor must explain to command that NT doesn't want command treated here. There's very little if any in-universe reason as to why the guidelines would be this strict, and likewise it's very out of character for a medical role to refuse to treat an injured person.

Someone gets shot near sec, and they can't opportunistically get healed at brig strictly to give antags more of a chance, because it's too 'meta', that's a very immersion breaking situation.

 instead of that mess make command sop say that they should seek treatment at medical and not sec or something.
 
 the realistic solution here is to add the 'lax' sop, play a couple rounds, see if it actually becomes a meta/powergaming nightmare and if so change it to be more strict later that's it.
 
 > didn't i already write like 30 sentences why giving sec their personal healer is bad. from antag chances to metagaming.
 
if brigmed's only actual concern is treating prisoners and nothing else there isn't a point to the role. i can see why the name was changed to corpsman elsewhere.

please do feel free to elaborate i can't see 30 sentences somebody mentioned antag opportunities once. **i'm not being facetious when i ask for perspective.** Why is giving sec opportunity to recuperate safely, or have someone at the backline getting them up in crisis THAT bad? Sec is getting clowned on recently so I don't believe it's a power-level issue?
 
 the one brigmedic cannot treat the whole of sec by themselves. in most scenarios they can't treat poison, radiation, cellular, genetic, missing limbs, severe limb damage, gibbing, hollow bodies, and however many other things. It's one guy with topicals. 
 
medical isn't getting sidelined by one person, and hey, if a dead officer requires more than a bandage to heal, now you have a security role, which has been stripped of most ways to defend themselves at this point, entering medical to use their equipment! They might even be distracted treating this officer.

besides brigmed isn't out on the field as a secoff. by the time a secoff is crit and being dragged somewhere they've been stripped clean anyway. 

Is the issue here that non-sec can't treat sec, that sec might be treated in a restricted area, or both?

> AAAAAAAAAAA

The brigmedic, a role that is both security and medical, which apparently can't do security, nor medical. Great. 

Fine your opinion is the role shouldn't exist, let's pretend it does exist. How do you change it in order to make it work? Taking away everything as said leaves them as a 2nd warden.

HenhenIII commented:
> the strict guidelines are already written in the rules. no powergaming is a rule. if somebody isn't respecting that adding sop isn't gonna make a difference.

It infact will make a huge difference. Powergaming for every role is different, it's need clarification or people could argue that X thing isn't powergaming because it isn't mentioned in the SOP. There needs to be a line between powergaming and not powergaming or we are gonna have another problem of admins banning you for something that people generally think isn't powergaming and ect.

> why the fuck is command going to brigmed anyway they have BSO who's whole role is to ensure their survival. BSO gets meds and a surgery bag. Maybe i'm out of the loop but i don't see many command flocking to BSO to avoid stepping a foot in medical.

Now this is what I call a good argument but there's still flaws. Most goob players are lazy as fuck. BSO could be anywhere, everyone gets frustrated at calling someone on the radio and them not responding or taking ages to get there. So going to made despite the danger is worth it because it saves 4 times the amount of time than waiting or searching for them. It's in human nature that being angry or frustrated has a bigger effect than having joy or whatever the people with "must stop antags at all cost" get. In the other hand the Brigmedic is always in sec, there's no need to search them, no need to call them. They will be almost always in security. 

> adding arbitrary hard restrictions preventing someone from treating does more harm than good. A hurt command member steps into sec where the medical role can't bandage them but the secoff can. preventing treatment solves nothing, and makes for an awkward in-universe exchange where the doctor must explain to command that NT doesn't want command treated here. There's very little if any in-universe reason as to why the guidelines would be this strict, and likewise it's very out of character for a medical role to refuse to treat an injured person.

Secoffs can't go around healing command, that's not part of their job, you could argue that "why passenger could heal command but sec can't". The difference is that the passenger gains nothing from healing command out of free will but security could have saved them from being jumped because they got low health of stuff being stolen off them while they are in med getting healed. The awkward situation you are talking about doesn't exist, there's a good IC reason for brigmedic not healing command nor sec, "medicine should be preserved for prisoners and not wasted for minor cuts and etc." like cargo shouldn't be going around healing people, shouldn't the brigmedic go around healing command. "Out of character" there's exceptions like salvager are part of cargo but they salvage instead of going around the station, collecting shit to sell, brigmedic is a medical officer but not the kind to heal everyone.

> Someone gets shot near sec, and they can't opportunistically get healed at brig strictly to give antags more of a chance, because it's too 'meta', that's a very immersion breaking situation.

Tf you tweaking about? Getting shot is an emergency situation, usually when people are shot, it's deadly meaning that they are gonna get more bullets than 1. That makes it an emergency situation. I litterly defined what an emergency would be a few messages ago. If it was a single bullet, he can walk it off and go to med.

> if brigmed's only actual concern is treating prisoners and nothing else there isn't a point to the role.

There, you reached to my hole point why this role is not needed.

> please do feel free to elaborate i can't see 30 sentences somebody mentioned antag opportunities once. i'm not being facetious when i ask for perspective.

I can't find it right now but I was on about a rant that "secoffs are always valid meaning they are gonna be the most hurt by antags, meaning they make up about 40% of patients. There might be shifts where medical only gets non secoffs but by in-game logic medical having a job heavily relays on antags doing shit meaning sec is gonna intervene and likely get hurt." Not this detailed but had the same point. There's also the fact that medical is the second most dangerous antag play spot. Since a lot of antag gameplay surrounds it, from killing to keeping them alive. It would make secoff not go to med often leaving it more in danger than it needs to be.

> Why is giving sec opportunity to recuperate safely, or have someone at the backline getting them up in crisis THAT bad? Sec is getting clowned on recently so I don't believe it's a power-level issue?

I'm gonna be the one who starts to tweak reading these soon. CRISIS = EMERGENCY, SAME THING. Also sec getting clowned on is mostly a skill issue. They just need to get robuster, they got 1 hit batons and instant judo stuns, just blatant skill issue. 

> the one brigmedic cannot treat the whole of sec by themselves. in most scenarios they can't treat poison, radiation, cellular, genetic, missing limbs, severe limb damage, gibbing, hollow bodies, and however many other things. It's one guy with topicals.

Exactly, that's why it should be saved for the prisoners and not wasted on other people (if not emergency).

> medical isn't getting sidelined by one person, and hey, if a dead officer requires more than a bandage to heal, now you have a security role, which has been stripped of most ways to defend themselves at this point, entering medical to use their equipment! They might even be distracted treating this officer.

"Dead officer"  EMERGENCY, DEFINED AS "UNABLE TO GET TO MED" 👹👹👹

> The brigmedic, a role that is both security and medical, which apparently can't do security, nor medical. Great.

Can't do security since it would be "secoff with med acces", restricted to do medical only on prisoners because all the reasons stated above.

> Fine your opinion is the role shouldn't exist, let's pretend it does exist. How do you change it in order to make it work? Taking away everything as said leaves them as a 2nd warden.

That's what I call either let them be super power gamy and meta gamy or restrict them into a sitting duck, there isn't a middle point. This role just doesn't work. If may work on paper as "concept" aka the idea seems to fit but if we get down to it, it doesn't.

Martynas6ha4 commented:
> This role just doesn't work.

If it didn't work at all, people wouldn't play it. While I've not been as active rn due to uni, even in the couple rounds I was in goob,  there were distinct folk going out of their way to hop to play as brigmed.

I get it's not fully representative for the role given the lack of protocol and player bias, with an added role you get a lot more shitters that can abuse it. Despite that, there is love for the role and I think it's worth at least trying.

> emergency situation(s)

Okay good news we share viewpoints somewhere sometimes.

And I'll admit i straight up fucking forgot about the damn emergency clause. Wow I'm moronic.

The issue is that even with that, it's ambiguous. I would like to think that *generally* people can pick up on what is an emergency, and what isn't.

however my dumbass accentuates that foolproof measures create bigger fools:

> "Dead officer" EMERGENCY

ignoring the fact i forgot that brigmed can treat in an emergency, - It still didn't really jump out at me that a dead secoff, could be a fucking emergency (i know right). I don't know, maybe i had too much 'brigmed cant treat sec' on my mind. I figured that a dead secoff, in that situation, isn't gonna get much dead-er (besides rotting lmao) and med could deal with them if the brigmed wasn't.

But in that mess i was mentioning a point about sec still kinda being at med for more serious injuries though i'll get on the secmed issue later.

> EMERGENCY, DEFINED AS "UNABLE TO GET TO MED"

I think it's a very good catch-all way of wording it. But still, I dislike the idea that there should be doubt whether a role is allowed to treat someone. 

No brigmed shouldn't go out as a paramed healing every tider. Yet I believe they should have agency as to where their limited resources go.

>  If it was a single bullet, he can walk it off and go to med.

Makes sense, fits the criteria as 'non-emergency',  yet in an area with supplies, a doctor, and wounds to treat, it's near instinctual to me to in fact, help. Maybe I need to rewire my brain on that.

I feel that it's common decency to at least spare a bandage if somebody is hurt. Brigmed gets a medhud and if someone is low, happens to be within sec due to circumstance, and is important personnel like command, it is just unsafe to leave them as-is despite them being able to get to med.

I get that's sort of the point - they go to med and that way there's a chance for antags anywhere between sec and med or in med against them. Also gives sec opportunity to act upon that. Yet anyone with situational awareness may opt to use two bandages and avoid the risk. It's rational thought, despite breaking the supposed sop.

It's an engineered edge-case situation fair, yet in this case the argument of preserving resources doesn't pass for me. I suppose the point I'm trying to make is that it's the nature of medical personnel to help people. 

I'll admit i just straight up don't have an argument for this one beyond immersion. Best I can say is that it takes away role gameplay. There is an in-between solution but i've not found it yet.

> Sec healing and medical hotzone

> It would make secoff not go to med often leaving it more in danger than it needs to be.

That's a good argument that I hadn't considered and I honestly don't have a counter for it.

I can argue on the sec giving med work bit I think that it wouldn't make med jobless, but I hadn't thought about sec being there by happenstance, (to heal or whatever) and stuff going down that they may deal with.

next best i can come up with as a "response" is that if sec is allowed to be healed then it isn't as much of a takeaway from antags since they will be less at medical thus letting em do more. Then again, that can also be a bad thing since sec & antag clashes bring upon more interesting gameplay.

It also sucks since not allowing brigmed to heal sec beyond emergencies takes away a lot of gameplay. brigmed isn't a secoff, yet part of the appeal is being there for sec too. darn.

I got nothin rn, I'm gonna let the ideas simmer for a bit and see if i can cook. I wonder how deltav or such handle that part specifically if even. It sounds quite significant for what it is maybe it's a bigger issue for LRP since there's more stuff happening.

HenhenIII commented:
> I feel that it's common decency to at least spare a bandage if somebody is hurt. Brigmed gets a medhud and if someone is low, happens to be within sec due to circumstance, and is important personnel like command, it is just unsafe to leave them as-is despite them being able to get to med.

Now i figured it out. You are trying to "make the role as good/useful as possible". brigmedic doesn't need to have the solution for everything. you can also call "BSO, mind following X command to med, their hurt" or "hey secoff, can you follow them to med. they are hurt".

gluesniffler commented:
good god what is all of the mald going on here, bros it aint that deep, go outside

anyway fix merg conflicts later this week pls, bunch of map PRs coming in and you'll suffer because of it :trollface:

HenhenIII commented:
> good god what is all of the mald going on here, bros it aint that deep, go outside

- see good arguments
- have no good arguments in response to them
- call it mald

Average GitHub experience.

PunishedJoe commented:
People already ask hop/hos ect to become brigmed in game, making it a job will just make it easier for people who want to be brigmed to be brigmed, not having the job doesn't remove the demand nor does it mean nobody will ever make the job in a round

deltanedas commented:
just roleban people if they abuse it same as everything (rolebanning shitters unheard of on goob :trollface:)

HenhenIII commented:
> just roleban people if they abuse it same as everything (rolebanning shitters unheard of on goob :trollface:)

Please tell me you put thought into this actually. Ever thought that abuse needs to be defined because otherwise people will stretch things for their own personal gain. This WHOLE discussing been about defining abuse and not abuse.

Aidenkrz commented:
Hasn't been touched outside of people arguing in over a month.
